project1/src/affinty_utils.h

```diff
@@ -0,0 +1,13 @@
+#ifdef WITH_CORE_AFFINITY
+
```

Comment:
Yu are missing the #define to ensure that this is not run multiple times because C is annoying

Reply:
Should be fixed now with the solution we talked about at school